arch/arm/core/cortex_m/ram_vector_table.ld

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2025 Silicon Laboratories Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Vector table is not necessarily at the start of the ram region: in the case that you also use
+ * code rellocation, it will place the vector table after the rellocated code. Rellocated code
+ * will always be at the start of the RAM SECTION because of the hardcoded generated value in file
+ * with "gen_relocate_app.py". It can create quite big gap and lose about the size of alignment
+ * of RAM Space
+ */
+
+SECTION_PROLOGUE(.sram_vt,,)
+{
+/* Heritage of vector table alignment in flash (see vector_table.ld in arch/arm/core/) */
+#if defined(CONFIG_ARMV6_M_ARMV8_M_BASELINE)
+. = ALIGN( 1 << LOG2CEIL(4 * 64) );
+#elif defined(CONFIG_ARMV7_M_ARMV8_M_MAINLINE)
+. = ALIGN( 1 << LOG2CEIL(4 * 32) );
```

Comment:
It's generally preferred to put the alignment onto the chunk in the object file instead of manually here.  Is that not already done?